pandas/tests/indexes/interval/test_base.py

```diff
@@ -86,3 +86,15 @@ def test_getitem_2d_deprecated(self):
         with pytest.raises(ValueError, match="multi-dimensional indexing not allowed"):
             with tm.assert_produces_warning(DeprecationWarning, check_stacklevel=False):
                 idx[:, None]
+
+    def test_get_indexer_non_monotonic(self):
```

Comment:
Can you move this to `tests/indexes/interval/test_indexing.py` within the `TestGetIndexer` class? LGTM otherwise.